packages/shared/src/evm.ts

```diff
@@ -1,8 +1,13 @@
 import { Signer } from 'ethers';
 import { getEvmChainId } from './consts';
-import { Chain, ChainflipNetwork } from './enums';
+import { Chain, ChainflipNetwork, ChainflipNetworks } from './enums';
 import { assert } from './guards';
 
+export const assertIsEvmChain = (chain: Chain) => {
+  const evmChainId = getEvmChainId(chain, ChainflipNetworks.backspin);
```

Comment:
Are we always supposed to check against `backspin` network?

Reply:
this kinda a shortcut to prevent explicitly adding the information if a chain is an evm chain. if the chain (eg `Arbitrum`) has an evm chain id on backspin, we can assume that it is an evm chain (on all networks). we dont care about what the id is, only if it is set